### DISCUSSION TITLE
Transitioning from Paddle payments to Proton-managed account

### DISCUSSION THREAD

**Discussion by gitnitwhit** [simple-login/app#1196] [Q&A] [answered]
Apologies if this has been answered, but I couldn't find it on the website or in the forum.

If I previously paid for SimpleLogin through Paddle, but I have Proton Unlimited, do I need to do anything to ensure that I don't accidentally pay through Paddle again when my renewal comes up?

Thank you!

**Comment by nguyenkims** [comment, marked as answer]
You can cancel the Paddle subscription in this case to avoid being charged next year/month.

**Reply by EnXevi** [reply]
To do this, I just have to cancel the subscription on Simplelogin settings or I have to do it on paddle?
Thanks!

**Comment by zoswizard** [comment]
I've connected my ProtonMail Visionary account to my Simplelogin account. Please confirm that once we cancel the subscription on Simplelogon it'll keep on working afterwards. The warning message is scary.

**Reply by EnXevi** [reply]
I've finally unsubscribed and all is working OK. It tells that my subscription is canceled but it also says that is controlled by Proton account

**Reply by zoswizard** [reply]
Thanks. Will try myself

**Reply by zoswizard** [reply]
I'm seeing this:

(Cancelled) Yearly plan subscribed via Paddle. [Manage Subscription)
Premium subscription managed by Proton.

**Reply by shinjipons** [reply]
I would like to have someone from SimpleLogin to confirm this. My renewal is next month and I have a Protonmail Unlimited account that is already connected to SimpleLogin.

**Reply by gitnitwhit** [reply]
So I think it is fine in this situation to cancel your membership through the settings page on SimpleLogin's site. When I did this, I got the message saying that my subscription through Paddle wouldn't renew, but when I clicked on "resubscribe", it said that I was already subscribed via my Proton subscription. Of course, it would be good to get confirmation that this is how things are working.

**Reply by nguyenkims** [reply]
I confirm that you can cancel the Paddle subscription in this case to avoid being charged next year/month.
